docs/en/index.rst

```diff
@@ -4,16 +4,40 @@
 Italian eIDAS Wallet Technical Specifications
 =============================================
 
-[TODO INTRO]
+Introduction
+------------
 
-Introduzione
+**What is eIDAS**
 
-cos'è eIDAS
+eIDAS stands for "electronic IDentification, Authentication and trust Services" and is the commonly used name for the EU Regulation, 910/2014, on electronic identification and trust services for electronic transactions in the internal market, repealing the old signature Directive 1999/93/EC.
 
-cos’è IT-Wallet
+The new eIDAS project establishes the European Digital Identity Wallet (EUDI Wallet) and proposes to overcome the dissimilarities, both in technological and user experience terms, that exist among the 21 digital identities (eIDs) currently active within as many as 16 Member States.
 
-scopo delle regole tecniche
+**What is IT Wallet**
 
+The IT Wallet project was created to improve the national digital identity governance experience and respond to the input received from the European community in the areas of innovation, decentralization and digital awareness. 
+
+To date, three identity systems coexist in Italy for access to public and private web services, namely: 
+
+ - Electronic Identity Card (CIE);
+ - Public Digital Identity System (SPID);
+ - Health Card - National Service Card (TS-CNS). 
+
+The result is a difficult, and thus costly, experience for citizens and service providers who must implement and maintain multiple authentication systems to ensure citizens' access to their digital services.
+
+Therefore, the IT Wallet proposes to:
+
+ - Streamline the digital identity ecosystem in Italy, optimizing the allocation of public resources;
+ - Simplify the digital access experience for citizens, the Public Administration and businesses;
+ - Integrate new functions related to certified attributes, strengthening the digital identity model and promoting the inclusion of public and private entities;
+ - Consolidate best practices in digital identities in Italy by maximizing deployment, quality of use and infrastructure provision;
+ - Adequately accommodate the European Digital Identity Wallet on the basis of the experience acquired by developing the national Wallet in full compliance with the European model.
+
+In order to achieve these objectives and enhance the already active and eIDAS-notified digital identity schemes, the IT Wallet project proposes a technological and governance evolution that envisages, in a progressive way, the migration of the digital identification component of CIE and SPID to IT Wallet.
```

Comment:
this text belongs to a strategical vision while this introduction is related to a technical reference

@ruphy WDYT?

Reply:
I like to have a short strategy intro to a tech document to put it in context, as long as it doesn't span more than a couple of pages. But these are just my two cents!